tests/projects/models.py

```diff
@@ -171,6 +181,21 @@ def name_with_priority(self) -> str:
         return f"{self.kind}: {self.priority}"
 
 
+class Version(NamedModel):
```

Comment:
I tried not to further extend types hierarchy, but this one is required to cover #362 changes